CHANGELOG-1.1.md

```diff
@@ -1,3 +1,62 @@
+- Fix the PD `location-labels` configuration ([#1941](https://github.com/pingcap/tidb-operator/pull/1941), [@aylei](https://github.com/aylei))
+- Able to pause and unpause tidb cluster deployment via `spec.paused`. ([#2013](https://github.com/pingcap/tidb-operator/pull/2013), [@cofyc](https://github.com/cofyc))
+- Default the `max-size` and `max-backups` for tidb server configuration with `300` and `3` if the tidb cluster was deployed by CR. ([#2045](https://github.com/pingcap/tidb-operator/pull/2045), [@Yisaer](https://github.com/Yisaer))
+- Able to configure custom env for components. ([#2052](https://github.com/pingcap/tidb-operator/pull/2052), [@cofyc](https://github.com/cofyc))
+- Fix the error that `kubectl get tc` cannot show correct images ([#2031](https://github.com/pingcap/tidb-operator/pull/2031), [@Yisaer](https://github.com/Yisaer))
+- 1. Default the `spec.tikv.maxfailureOver` and `spec.tidb.maxfailureOver` to 3 when they are no defined
+  2. Change the action that when `maxfailureOver` is zero, operator would not failover instead of creating limitless instances. ([#2015](https://github.com/pingcap/tidb-operator/pull/2015), [@Yisaer](https://github.com/Yisaer))
+- Support deploying TiDB Cluster with TidbCluster and TidbMonitor CRs via Terraform on ACK. ([#2012](https://github.com/pingcap/tidb-operator/pull/2012), [@DanielZhangQD](https://github.com/DanielZhangQD))
+- Update PDConfig for TidbCluster to pd v3.1.0 ([#1928](https://github.com/pingcap/tidb-operator/pull/1928), [@Yisaer](https://github.com/Yisaer))
+- Support deploying TiDB Cluster with TidbCluster and TidbMonitor CRs via Terraform on AWS. ([#2004](https://github.com/pingcap/tidb-operator/pull/2004), [@DanielZhangQD](https://github.com/DanielZhangQD))
+- Update TidbConfig for TidbCluster to tidb v3.1.0 ([#1906](https://github.com/pingcap/tidb-operator/pull/1906), [@Yisaer](https://github.com/Yisaer))
+- Allow users to define resources for initContainers in TiDB initializer job. ([#1938](https://github.com/pingcap/tidb-operator/pull/1938), [@tfulcrand](https://github.com/tfulcrand))
+- TLS support for Pump and Drainer ([#1979](https://github.com/pingcap/tidb-operator/pull/1979), [@weekface](https://github.com/weekface))
+- Add doc and examples for auto-scaler and intializer ([#1772](https://github.com/pingcap/tidb-operator/pull/1772), [@Yisaer](https://github.com/Yisaer))
+- 1. Add check to guarantee the NodePort won't be changed if the serviceType of TidbMonitor is NodePort
+  2. Add EnvVar sort to avoid the monitor rendering different results from the same TidbMonitor spec.
+  3. Fix the problem that Tidbmonitor LoadBalance IP is not used. ([#1962](https://github.com/pingcap/tidb-operator/pull/1962), [@Yisaer](https://github.com/Yisaer))
+- Make tidb-initializer support TLS ([#1931](https://github.com/pingcap/tidb-operator/pull/1931), [@weekface](https://github.com/weekface))
+- 1. Fix the problem that Advanced StatefulSet could not work with Webhook
```

Comment:
```suggestion
  1. Fix the problem that Advanced StatefulSet could not work with Webhook
```

Reply:
this release note contains multiple items, no need to drop `-`.